### ISSUE TITLE
Advice on Establishing my first connection

### ISSUE THREAD

**Issue by risingmole** [CongducPham/LowCostLoRaGw#39]
Hi There, 
Firstly thanks for your great work on this project! I am really excited about learning how to use LoRa for my own projects!

I have currently set up your lora_gateway_pi2 code running on a RPi3 B - with a RFM95W chip and a simple wire antenna. For my node I have an arduino uno (again attached via spi to a RFM95W chip). 

The code seems to run ok on the RPi, however I am yet to get message across from one board to the other. 

From the Arduino end I get this via the serial monitor (running simple temp): 

Get back previous sx1272 config
Using packet sequence number of 1
Setting Mode: state 1
Setting Channel: state 1
Setting Power: state 1
Setting node addr: state 1
SX1272 successfully configured
Reading 534
(Temp is 260.74
Sending \!#4#260.74
Real payload size is 11
--> CAD duration 300
###1
--> waiting for 1 DIFS (DIFS=3SIFS) = 0
--> retry
--> CAD duration 300
###1
--> waiting for 1 DIFS (DIFS=3SIFS) = 0
--> retry
--> CAD duration 300
###1
--> waiting for 3 DIFS (DIFS=3SIFS) = 0
--> retry
--> CAD duration 301
###1
--> waiting for 7 DIFS (DIFS=3SIFS) = 0
--> retry
--> CAD duration 300
###1
--> waiting for 7 DIFS (DIFS=3SIFS) = 0
--> retry
--> CAD duration 300
###1

It then remains stuck in a loop waiting for different DIFS. 

Have you encountered this issue before? Is it likely an issue with my wire antennas? Or with the channels not matching?

As an aside, I am based in Australia, so aiming to use the 915MHz band. 
I am not sure if I have set this up correctly. On the arduino code I have uncommented BAND900. I have set the channel to CH_12_900. I have left the Both modules are using LORAMODE = 1. 

On the Raspberry Pi I have edited the lora_gateway.cpp file to uncomment the DEFINE of BAND900. I am not sure exactly how to force the band to CH_12_900 on the RPi??

Any advice appreciated! 

Cheers

**Comment by risingmole** [comment]
In particular, one issue that I appear to be having is that despite commenting out the line of DEFINE BAND868,  my terminal output is 
pi@raspberrypi:~ $ pwd
/home/pi
pi@raspberrypi:~ $ cd lora_gateway
pi@raspberrypi:~/lora_gateway $ sudo ./lora_gateway_pi2
SX1276 detected, starting.
SX1276 LF/HF calibration
...
^$**********Power ON: state 0
^$Default sync word: 0x12
^$LoRa mode 1
^$Setting mode: state 0
**^$Channel CH_10_868: state 0**
^$Set LoRa Power to M
^$Power: state 0
^$Get Preamble Length: state 0
^$Preamble Length: 8
^$LoRa addr 1: state 0
^$SX1272/76 configured as LR-BS. Waiting RF input for transparent RF-serial bridge

It is my understanding that it should not be in **Channel CH_10_868:** is BAND868 is not defined?

**Comment by risingmole** [comment]
A small update, I have resolved the issue with uncommenting BAND900 not seeming to make any difference. It was due to me not remaking the lora_gateway_pi2 file between edits to the .cpp file. Once I fixed that I did manage to get a packet through!  (but I have not been successful in sending any more). 

I have a question re the Arduino side of things: my output is this: 

> Using packet sequence number of 23
> Setting Mode: state 1
> Setting Channel: state 1
> Setting Power: state 1
> Setting node addr: state 1
> SX1272 successfully configured

does state 1 mean unsuccessful? If so, how can the whole SX1272 be successfully configured?
